Dockerfile

```diff
@@ -0,0 +1,44 @@
+# syntax=docker/dockerfile:1.4
+
+# Use Ubuntu 22.04 as the build environment
+FROM ubuntu:22.04 as build-environment
```

Comment:
`FROM rust:bookworm as builder`
would save you some installs

Reply:
Done